thirdparty/github.go

```diff
@@ -1099,34 +1052,43 @@ func GetGithubTokenUser(ctx context.Context, token string, requiredOrg string) (
 	}, isMember, err
 }
 
-// CheckGithubAPILimit queries Github for the number of API requests remaining
-func CheckGithubAPILimit(ctx context.Context) (int64, error) {
-	caller := "CheckGithubAPILimit"
+// GetGithubAPILimit queries Github for all the API rate limits
+func GetGithubAPILimit(ctx context.Context) (*github.RateLimits, error) {
+	caller := "GetGithubAPILimit"
```

Comment:
why rename this caller / function ? this might break some of our splunk dashboards

Reply:
(I think it's okay to call this the same thing as the cron function since it's a different package; you could also change that cron job to be logGithubAPILimit like it was before)

Reply:
one of the reasons was so I didn't have to change the original CheckGithubAPILimit because it returns an int where I wanted the whole object. 

I updated it to keep the same name and caller though!